pkg/dump/dump.go

```diff
@@ -57,6 +58,11 @@ type Config struct {
 	// the intermediate state so that foreign key lookups will work.
 	// Partial applies to configure a partial state.
 	IsPartialApply bool
+
+	// This value is set via a flag in deck
```

Comment:
`go-database-reconciler` is also used by KIC, so potentially decK is not the only way to use this flag. I'd remove this reference.